.github/workflows/build-wheel-macos-arm64.yaml

```diff
@@ -340,6 +340,19 @@ jobs:
 
         cmake --build quantum-build --target check-dialects catalyst-cli
 
+    - name: Build Standalone Plugin
+      run: |
+        cmake -B standalone/build -G Ninja \
+            -DMLIR_DIR=$GITHUB_WORKSPACE/llvm-build/lib/cmake/mlir \
+            -DLLVM_EXTERNAL_LIT=$GITHUB_WORKSPACE/llvm-build/bin/llvm-lit \
+            -DCATALYST_TOOLS_DIR=$GITHUB_WORKSPACE/quantum-build \
+            -DPython3_EXECUTABLE=$(which python${{ matrix.python_version }}) \
+            -DPython3_NumPy_INCLUDE_DIRS=$(python${{ matrix.python_version }} -c "import numpy as np; print(np.get_include())") \
+            standalone
+
+        cmake --build standalone/build --target check-standalone
```

Comment:
We can't use the make recipe on mac?

Reply:
Here somehow these last two statements are needed:

```
            -DPython3_EXECUTABLE=$(which python${{ matrix.python_version }}) \
            -DPython3_NumPy_INCLUDE_DIRS=$(python${{ matrix.python_version }} -c "import numpy as np; print(np.get_include())") \
```

I suspect it has more to do with multiple versions of python being installed than it being a mac. Something that I also wouldn't expect to be necessary but probably somewhere else can be fixed. I could add these to the Makefile and rely on the PYTHON variable there. I would suggest to fix it in a follow up PR, because the bottleneck in testing issues here is the cache. Once these changes with VISIBILITY=default are enabled, testing becomes way easier / faster.

Reply:
> could add these to the Makefile and rely on the PYTHON variable there.

I think that's the way. In fact, didn't @joeycarter just do the same thing?

Reply:
Looks like he added it to all the wheel recipes and runtime make file:
https://github.com/PennyLaneAI/catalyst/pull/1293/files

> I would suggest to fix it in a follow up PR, because the bottleneck in testing issues here is the cache. Once these changes with VISIBILITY=default are enabled, testing becomes way easier / faster.

Sure if you think that's more practical.

Reply:
Yes I added something similar in my PR to make sure we use a specific version of Python when building, since a system might have multiple versions installed. Although mine was a little different since it was to direct the `find_package(python)` CMake command required by nanobind.

Reply:
https://github.com/PennyLaneAI/catalyst/pull/1287/commits/79434f63008fffd638a94a2244b2819a8630b123